pennylane/gradients/general_shift_rules.py

```diff
@@ -388,48 +375,12 @@ def generate_multi_shift_rule(frequencies, shifts=None, orders=None):
 
     for f, s, o in zip(frequencies, shifts, orders):
         rule = generate_shift_rule(f, shifts=s, order=o)
-        rules.append(process_shifts(rule).T)
+        rules.append(process_shifts(rule))
 
     return _combine_shift_rules(rules)
 
 
-def generate_shifted_tapes(tape, idx, shifts, multipliers=None):
-    r"""Generate a list of tapes where a marked trainable parameter
-    has been shifted by the values given.
-
-    Args:
-        tape (.QuantumTape): input quantum tape
-        idx (int): index of the trainable parameter to shift
-        shifts (Sequence[float or int]): sequence of shift values
-        multipliers (Sequence[float or int]): Sequence of multiplier values to
-            scale the parameter by before shifting. If provided has to have the
-            same length as ``shifts``. If not provided, the parameter will not
-            be scaled.
-
-    Returns:
-        list[QuantumTape]: List of quantum tapes. Each tape has parameter
-        ``idx`` shifted by consecutive values of ``shift``. The length
-        of the returned list of tapes will match the length of ``shifts``.
-    """
-    params = list(tape.get_parameters())
-    tapes = []
-
-    for i, s in enumerate(shifts):
-        new_params = params.copy()
-        shifted_tape = tape.copy(copy_operations=True)
-
-        if multipliers is not None:
-            m = multipliers[i]
-            new_params[idx] = new_params[idx] * qml.math.convert_like(m, new_params[idx])
-
-        new_params[idx] = new_params[idx] + qml.math.convert_like(s, new_params[idx])
-        shifted_tape.set_parameters(new_params)
-        tapes.append(shifted_tape)
-
-    return tapes
-
-
-def generate_multishifted_tapes(tape, indices, shifts, multipliers=None):
+def generate_shifted_tapes(tape, indices, shifts, multipliers=None):
     r"""Generate a list of tapes where multiple marked trainable parameters
     have been shifted simultaneously by the values given.

```

Comment:
Since you have merged the functions, do you need to update the docstring?

Reply:
Fair enough, thanks! :)